tron/core/action.py

```diff
@@ -53,6 +54,7 @@ def from_config(cls, config):
             executor=config.executor,
             cpus=config.cpus,
             mem=config.mem,
+            disk=config.disk or 1024.,
```

Comment:
this is probably fine, but in python `0.0` is `false`, so user specifying `disk=0` will erroneously get `disk=1024`